src/hydra_zen/structured_configs/_add_conf.py

```diff
@@ -255,53 +304,12 @@ def check_signature_has_defaults_for_all_parameters(cls_or_func):
                 defaultless_params.append(k)
 
 
-def get_zen_meta_defaults(zen_meta=None):
-    """> It returns a dictionary of default values for the `zen_meta` dictionary.
-
-    Args:
-        zen_meta (dict): a dictionary of zen_meta parameters
-
-    Returns:
-        A dictionary with the keys: name_, group_, package_, provider_, defaults.
-    """
-    _zen_meta_defaults = dict(
-        name_=None,
-        group_=None,
-        package_="_group_",
-        provider_=None,
-        defaults=["_self_"],
-    )
-    if zen_meta is None:
-        return _zen_meta_defaults
-    else:
-        _zen_meta_defaults.update(zen_meta)
-        return _zen_meta_defaults
-
-
-def merge_zen_meta_defaults(name_, group_, package_, provider_, defaults):
-    """It merges the zen_meta dictionary with the defaults dictionary.
-
-    Args:
-        name_: The name of the resource.
-        group_: The group of the resource.
-        package_: The name of the package that the resource belongs to.
-        provider_: The name of the provider.
-        defaults: The default values for the parameters.
-
-    Returns:
-        The zen_meta dictionary is being returned.
-    """
-    zen_meta = get_zen_meta_defaults()
-    zen_meta["name_"] = zen_meta.get("name") or name_
-    zen_meta["group_"] = zen_meta.get("group") or group_
-    zen_meta["package_"] = zen_meta.get("package") or package_
-    zen_meta["provider_"] = zen_meta.get("provider") or provider_
-    zen_meta["defaults"] = zen_meta.get("defaults") if defaults is None else defaults
-    return zen_meta
+global cs
+cs = ConfigStore.instance()
 
 
 @dataclass
-class BatteriesIncludedConf:
+class ZenExtras:
```

Comment:
we should talk about this, it looks like some of the changes here are undoing some purposeful refactoring I put in place